### PR TITLE
CORE-6791: Simulator API KDocs

### PR DESCRIPTION
**Pull request by LizR3**

### CODE REVIEW

**Review by mattbradburyr3 — DISMISSED** [review]
Approved - non code changes

File: simulator/api/build.gradle [hunk 1]
```diff
@@ -21,9 +22,12 @@ dependencies {
 
     testRuntimeOnly "org.slf4j:slf4j-simple:$slf4jVersion"
 
-
+    dokkaHtmlPlugin "org.jetbrains.dokka:kotlin-as-java-plugin:$kotlinVersion"
 }
 
 tasks.named('jar', Jar) {
```
Comment (ronanbrowne) [inline simulator/api/build.gradle:28]: I suspect this is not what you want. Your overwriting the Jar task to produce a java doc  jar.

you want something like this....

https://github.com/corda/corda-api/blob/release/os/5.0/build.gradle#L199-L207

it's a addental task not a replacement for the jar task

Comment (LizR3) [reply simulator/api/build.gradle:28]: Thank you!

**Comment by mcgovc** [comment]
Necessary build changes should be covered in https://github.com/corda/corda-runtime-os/pull/2171 once merged. 
Doc changes also made in https://github.com/corda/corda-runtime-os/pull/2168 (approved by BLT, but should Docs also review?) 

This one can probably be closed off?

**Comment by LizR3** [comment]
Closing as replaced by a PR that just has docs etc. without build changes.

**Pull request closed without merging**